pokemongo_bot/cell_workers/move_to_map_pokemon.py

```diff
@@ -245,7 +245,7 @@ def snipe(self, pokemon):
                     exists = True
 
                     # Also, if the IDs arent valid, update them!
-                    if not pokemon['encounter_id'] or not pokemon['spawnpoint_id']:
+                    if not pokemon['encounter_id'] or not pokemon['spawn_point_id']:
```

Comment:
Social info does not have the spawnpoint ID, only the encounter. So under social mode, it would never hit that if, meaning that it would never fix the spawnpoint and catch the target. Pokemons without a spawnpoint are lured pokemons (but social pokemons arent really lured) and it would crash (error is suppressed) and skip. The "does not exist" is a failed attempt to find it nearby after teleporting to it.